up.go

```diff
@@ -29,6 +29,8 @@ import (
 	"os/exec"
 	"sync"
 
+	flag "github.com/spf13/pflag"
```

Comment:
Please don't rename pflag to flag. I understand they provide such an example in the docs, but the reasons they provide don't apply here, we can full well use the real name of the package.

Also please remove the empty line 33, there's no reason for this to stand out from other github dependencies.